code-challenges.js

```diff
@@ -20,9 +20,22 @@ const fruit2 = "banana"
 
 // --------------------1) Create the code that determines if a given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
-// Describe your process:
+//1> Describe your process:1>I determined to use conditional statements
+//2>created a variable of boilingPoint
+//3>declare the variable with the let command
+//4>console log the temperatures in the instructions.
+
 
-const temp = 42
+let boilingPoint = (42 , 350 , 212 )
```

Comment:
The logic looks good here, I think the main thing to consider is what you currently assigned to the boilingPoint variable vs the variable you are checking.  You could set a variable of boilingPoint to 212 and compare it against the given temp variables in your conditionals.